appveyor.yml

```diff
@@ -0,0 +1,15 @@
+version: '{build}'
+build_script:
+- cmd: >-
+    set RUBY_VERSION=200
+
+    set PATH=C:\Ruby%RUBY_VERSION%\bin;%PATH%
+
+    gem install bundler
+
+    bundle install
+
+    build.netstd.cmd
+test: off
+on_finish:
+    - ps: $root = Resolve-Path .\artifacts; [IO.Directory]::GetFiles($root.Path, '*.*', 'AllDirectories') | % { Push-AppveyorArtifact $_ -FileName $_.Substring($root.Path.Length + 1) -DeploymentName to-publish }

```

Comment:
Oh, one more thing. The build script should probably be split up into `init` and `install` sections, e.g.

```yaml
version: '{build}'
init:
- cmd: >-
    set RUBY_VERSION=200
    set PATH=C:\Ruby%RUBY_VERSION%\bin;%PATH%
environment:
  VERSION_SUFFIX: -netstd
install:
- cmd: >-
    gem install bundler
    bundle install
build_script:
- cmd: build.netstd.cmd
test: off
...
```

I guess `init` could also be moved to `environment`, e.b.

```yaml
version: '{build}'
environment:
  RUBY_VERSION: 200
  PATH: C:\Ruby%RUBY_VERSION%\bin;%PATH%
  VERSION_SUFFIX: -netstd
install:
- cmd: >-
    gem install bundler
    bundle install
build_script:
- cmd: build.netstd.cmd
test: off
...
```

(I'm not sure if setting the `PATH` variable that way will work.)

I guess it's debatable whether `gem install bundler` should be part of `init` or `install`. On that note, I wonder if the AppVeyor environment comes pre-installed with bundler? It seems Ruby is pre-installed, so it would probably make sense to include bundler by default.